core/Makefile

```diff
@@ -34,14 +34,15 @@ include ../buildenv.mk
 App_Name := ehsm-core
 Enclave_Name := libenclave-$(App_Name).so
 Signed_Enclave_Name := libenclave-$(App_Name).signed.so
-
+Napi_Name := libehsmnapi.so
 Enclave_Config_File := Enclave/enclave.config.xml
 Enclave_Signing_Key := Enclave/enclave_private_test.pem
```

Comment:
change the commit msg title as 
“make the napi as a library for nodejs”

Reply:
It will be fix.